app/uk/gov/hmrc/tai/model/domain/benefits/Benefits.scala

```diff
@@ -42,6 +45,53 @@ case class CompanyCarBenefit(
 
 object CompanyCarBenefit {
   implicit val formats: OFormat[CompanyCarBenefit] = Json.format[CompanyCarBenefit]
+
+  def companyCarBenefitReads = new Reads[CompanyCarBenefit] {
```

Comment:
Do these need to be defs? Also could we have type annotations for these?

Reply:
Also, why have we got an implicit Formats AND a Reads for CompanyCarBenefits? What's the difference?

Reply:
It was probably there already. I would leave it for now.

Yes for the type annotations